pkg/stub/handler.go

```diff
@@ -349,16 +350,25 @@ func (h *Handler) tbrInaccessible() bool {
 		}
 	}
 	err = wait.PollImmediate(5*time.Second, 3*time.Minute, func() (bool, error) {
-		tlsConf := &tls.Config{}
-		conn, err := tls.Dial("tcp", "registry.redhat.io:443", tlsConf)
+		// we have seen cases in the field with disconnected cluster where the default connection timeout can be
+		// very long (15 minutes in one case); so we do an initial non-tls connection were we can specify a quicker
+		// timeout to filter out that scenario and default to tbr inaccessible / Removed in an expedient fashion
+		connWithTimeout, err := net.DialTimeout("tcp", "registry.redhat.io:443", 15*time.Second)
```

Comment:
15 still a best guest number that might change in the future. is this something we can make configurable so we can have the customer change if it does not work for them?

Reply:
good question @dperaza4dustbit 

config changes where we have to change the config object and add a new field and expensive / costly

admittedly a "gut feeling", but for what we are dealing with here, it would be better to just have a hard coded value that is sufficient

at most, maybe add an environment variable on the deployment that could be read

minimally, I would be agreeable to running say the e2e-aws-operator test suite repeated times (maybe a dozen) to get a warmer fuzzy 

/test e2e-aws-operator

Reply:
OK understand @gabemontero , maybe take me through the process of making a config change to get a feeling on the price there.

Reply:
yep let's add that to Thursday's call, along with the cross referencing CI flakes with sippy that I mentioned in your openshift/origin PR